File: Tasks/Python Work/switch_case_for_choosing_operators_for_numbers.py
number_of_correct = 0
number_of_wrong = 0
for counter in range (5):

	number_one = int(input("Enter the first number: "))
	number_two = int(input("Enter the second number: "))
	operator = input("Enter the mathematical sign: ")

	print("Question: ",number_one, operator, number_two)

	correct_answer = 0

	match(operator):

		case'+': correct_answer = number_one + number_two;

		case'-': correct_answer = number_one - number_two;

		case'x': correct_answer = number_one * number_two;

		case'/': correct_answer = number_one / number_two;

	answer = float(input("Answer: "))

	if correct_answer == answer:
		print("You are correct")
		number_of_correct += 1

	else:
		print("You are wrong")
		number_of_wrong += 1

print("The number of correct answers = ",number_of_correct)
print("The number of wrong answers = ",number_of_wrong)


	

File: Tasks/Java Work/SwitchCaseForChoosingOperatorsForNumbers.java
import java.util.Scanner;
	
public class SwitchCaseForChoosingOperatorsForNumbers{
	public static void main(String[] args){
	Scanner input = new Scanner(System.in);

int numberOfCorrectAnswers = 0;
int numberOfWrongAnswers = 0;

for(int counter = 1;counter <= 5; counter ++){

System.out.println("Enter the first number");
int numberOne = input.nextInt();

System.out.println("Enter the second number");
int numberTwo = input.nextInt();

System.out.println("Enter the mathematical sign");
char operator = input.next().charAt(0);

System.out.printf("Question: %d %c %d%n",numberOne, operator, numberTwo);

double correctAnswer = 0;

switch(operator){

case'+': correctAnswer = numberOne + numberTwo;
	break;

case'-': correctAnswer = numberOne - numberTwo;
	break;

case'x': correctAnswer = numberOne * numberTwo;
	break;

case'/': correctAnswer = numberOne / numberTwo;
	
}

System.out.printf("Answer: ");
double answer = input.nextDouble();

if (correctAnswer == answer){
System.out.println("Your answer is correct");
numberOfCorrectAnswers ++; }

else {
System.out.println("Your answer is wrong");
numberOfWrongAnswers ++;  }

}

System.out.println("The number of correct answers = " + numberOfCorrectAnswers);
System.out.println("The number of wrong answers = " + numberOfWrongAnswers);

}
}




